.github/workflows/ci.yml

```
@@ -4,7 +4,8 @@ on:
   push:
     branches: [ "main" ]    
   pull_request:
-    branches: [ "main" ]    
+    branches: [ "main" ] 
```

yamllint

[error] too many spaces inside brackets

(brackets)